Documentation/FAQ.md

```diff
@@ -24,4 +24,12 @@ If you can't figure out what to do, ask in the `#build-problems` channel on Disc
 
 ## Why don't you use `$LIBRARY` instead of implementing `$FEATURE` yourself?
 
-The SerenityOS project tries to maximize hackability, accountability, and fun(!) by implementing everything ourselves.
\ No newline at end of file
+The SerenityOS project tries to maximize hackability, accountability, and fun(!) by implementing everything ourselves.
+
+## Does SerenityOS have a package manager?
+
+No, SerenityOS does not have a package manager. The project uses a monorepo approach, meaning that all software is built in the same style and using the same tools. There is no reason to have something like a package manager because of this.
+
+*However* there are ports which can be found in the [Ports directory](https://github.com/SerenityOS/serenity/tree/master/Ports). A port is a piece of software that can optionally be installed, might have not been built by us but supports running on SerenityOS. They act quite similar to packages, coming with an install script each.
```

Comment:
To make it work online *and* locally (like in TextEditor :sunglasses:, *I think*):

```suggestion
*However* there are ports which can be found in the [Ports directory](../Ports). A port is a piece of software that can optionally be installed, might have not been built by us but supports running on SerenityOS. They act quite similar to packages, coming with an install script each.
```